### PR TITLE
GHES integration update

### PR DESCRIPTION
**Pull request by ashokirla**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +24,13 @@ The GitHub integration for Slack gives you and your teams full visibility into y
...
 ## Installing the GitHub integration for Slack
 ### Requirements
-This app officially supports GitHub.com (which includes our GitHub Enterprise cloud-hosted offering) and Slack.com, but the team plans to support GitHub Enterprise Server (our self-hosted product) in the future.
+#### GHEC - GA
+This app officially supports GitHub.com (which includes our GitHub Enterprise cloud-hosted offering) and Slack.com. 
+
+#### GHES - Private preview
+We are planning to rollout GHES integration with Slack.com as part of GHES 3.6.
+
+This will be a private preview and interested customers can enrol for the preview by creating an issue from [here](https://github.com/integrations/slack/issues/new). We will help you with the onboarding process.
```
Comment (apurva1112) [inline README.md:33]: nit: enrol -> enroll

File: README.md [hunk 2]
```diff
@@ -308,9 +307,21 @@ If you are on Slack Enterprise Grid and have multiple Slack workspaces in your o
 - If you have already installed GitHub app in one of the workspaces in the grid, you can now install the app at org level and manage it across the workspaces by going to your enterprise grid management console -> installed apps.
 <p align="left"><img width="500" alt="InstalledApps" src="docs/images/InstalledApps.PNG"></p>
 
+## Roadmap and upcoming features
+
+* Support for GHES - We are supporting GHES integration as a private preview with GHES 3.6. Based on the feedback, we will decide the GA timelines.
+* Issue create and manage capabilities - Provide capabilities to create and manage issues directly from your chat. We are aiming to rollout this support by September, 2022.
```
Comment (apurva1112) [inline README.md:313]: nit `September, 2022` to `September 2022` (comma removed).

File: README.md [hunk 3]
```diff
@@ -308,9 +307,21 @@ If you are on Slack Enterprise Grid and have multiple Slack workspaces in your o
...
+* GitHub Actions notifications and manage capabilities - Provide capabilities to run, view and manage workflows from chat. We are iming to rollout this support by October, 2022.
```
Comment (apurva1112) [inline README.md:314]: nit: iming to aiming

Comment (apurva1112) [reply README.md:314]: October, 2022 to October 2022 (removed comma)

File: README.md [hunk 4]
```diff
@@ -308,9 +307,21 @@ If you are on Slack Enterprise Grid and have multiple Slack workspaces in your o
...
+* Customer filters support to manage notifications - This is one of the top asked feature. Provide support to customize notifications by providing advaced filter capabilities. Timlines are to be decided.
```
Comment (apurva1112) [inline README.md:315]: Should `Customer filters` be corrected to `Custom filters` ?

File: README.md [hunk 5]
```diff
@@ -31,7 +24,13 @@ The GitHub integration for Slack gives you and your teams full visibility into y
 --------
 ## Installing the GitHub integration for Slack
 ### Requirements
-This app officially supports GitHub.com (which includes our GitHub Enterprise cloud-hosted offering) and Slack.com, but the team plans to support GitHub Enterprise Server (our self-hosted product) in the future.
+#### GHEC - GA
```
Comment (apurva1112) [inline README.md:27]: Can add this subheading as a part of index nested under requirements.

File: README.md [hunk 6]
```diff
@@ -31,7 +24,13 @@ The GitHub integration for Slack gives you and your teams full visibility into y
...
+This app officially supports GitHub.com (which includes our GitHub Enterprise cloud-hosted offering) and Slack.com. 
+
+#### GHES - Private preview
```
Comment (apurva1112) [inline README.md:30]: Can add this subheading as a part of index nested under requirements.

File: README.md (hunk 5 above)
Comment (ashokirla) [reply README.md:27]: I think we are not doing 2nd level nested listing in the index. So it should be okay.

File: README.md (hunk 6 above)
Comment (ashokirla) [reply README.md:30]: I think we are not doing 2nd level nested listing in the index. So it should be okay.

**Review by aasiddiq — APPROVED** [review]

**Pull request merged**